pandas/core/window/rolling.py

```diff
@@ -1351,8 +1352,21 @@ def apply_func(values, begin, end, min_periods, raw=raw):
 
         return apply_func
 
-    def sum(self, *args, **kwargs):
+    def sum(self, *args, engine=None, engine_kwargs=None, **kwargs):
         nv.validate_window_func("sum", args, kwargs)
+        if maybe_use_numba(engine):
+            if self.method == "table":
+                raise NotImplementedError("method='table' is not supported.")
```

Comment:
i c, so coming back in a followon to remove this condition

Reply:
Correct. https://github.com/numba/numba/issues/6610 I think blocks us from using table with these other methods

Reply:
can you not just make a lambda that takes a single arg and do this? (i agree it is cleaner to do this way but seems to block it needlessly

Reply:
Unfortunately no. A jitted function with any of the `np.nan*` aggregations and the `axis` argument is unsupported currently.

Reply:
wait *inside* the function? oh the translation is not implemented interesting. can you not just

```
def f(arr):
    return np.nansum(arr.T)
```

to mimic `np.nansum(arr, 1)`?

Reply:
Looks like all numba's `np.nan*` aggregation functions are written to return scalar

https://github.com/numba/numba/blob/master/numba/np/arraymath.py#L958